src/index.js

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line import/no-unresolved
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import addHomeCtn from './home';
+import addOrderCtn from './order';
+import addContactCtn from './contact';
+
+const contentElement = document.querySelector('#content');
+
+const mainHtml = `
+<nav id="navbar" class="navbar  fixed-top  navbar-expand-lg navbar-dark  bg-dark">
+  <a class="navbar-brand " href="#">JS Restaurant</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+    aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse ml-4" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item px-3 active" id="homeBtn">
+        <a class="nav-link" data-id="homeBtn" href="#data">Home </a>
+      </li>
+      <li class="nav-item px-3" id="orderBtn">
+        <a class="nav-link" data-id="orderBtn" href="#main"> Order Food </a>
+      </li>
+      <li class="nav-item px-3" id="contactBtn">
+        <a class="nav-link" data-id="contactBtn" href="#main"> Contact </a>
+      </li>
+    </ul>
+    <form class="form-inline my-2 my-lg-0">
+      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+    </form>
+  </div>
+</nav>
+<main class="container" id="main"></main>
```

Comment:
Your design has absolutely no problem on its own, but however, you did not respect the guidelines. There are some lessons the guidelines wants us to learn, that's why we should follow it.

![image](https://user-images.githubusercontent.com/31897434/86684686-af2e5580-bffa-11ea-863a-e7df4e949cc5.png)


In other words;
**You need to append each element not hard coding them.**
kindly make use of `document.createElement` and `document.appendChild`
Example: 
`div.appendChild(Navbar)`